kml/serdes_gpx.cpp

```diff
@@ -139,6 +140,38 @@ void GpxParser::ParseColor(std::string const & value)
   m_color = kml::ToRGBA(colorBytes[0], colorBytes[1], colorBytes[2], (char)255);
 }
 
+static std::unordered_map<std::string, std::string> const kGarminToHex = {
```

Comment:
Where did you find these colors?

Reply:
1. As this constant is used inside a function only, it's better to move it there.
2. Can string_view be used as a key and value?

Reply:
> Where did you find these colors?

List is available here: https://www8.garmin.com/xmlschemas/GpxExtensionsv3.xsd
And for hex mapping: https://help.locusmap.eu/topic/extend-garmin-gpx-compatibilty

Reply:
> 1. As this constant is used inside a function only, it's better to move it there.

ok, I'll move it
> 
>     2. Can string_view be used as a key and value?

it is possible, but in this case we will need to convert it to std:string to pass it to GpxParser::ParseColor (this one accepts string because it is called from GpxParser::CharData that accepts string.
So one conversion will be required, in this case do we want to switch?

Reply:
Please add these links in the code here as a comment.